REUSE.toml

```diff
@@ -1,18 +1,28 @@
-# SPDX-FileCopyrightText: SAP SE
-# SPDX-License-Identifier: Apache-2.0
 version = 1
+SPDX-PackageName = "castellum"
+SPDX-PackageSupplier = "ospo@sap.com"
+SPDX-PackageDownloadLocation = "https://github.com/sapcc/castellum"
+SPDX-PackageComment = "The code in this project may include calls to APIs (\"API Calls\") of\n SAP or third-party products or services developed outside of this project\n (\"External Products\").\n \"APIs\" means application programming interfaces, as well as their respective\n specifications and implementing code that allows software to communicate with\n other software.\n API Calls to External Products are not licensed under the open source license\n that governs this project. The use of such API Calls and related External\n Products are subject to applicable additional agreements with the relevant\n provider of the External Products. In no event shall the open source license\n that governs this project grant any rights in or to any External Products,or\n alter, expand or supersede any terms of the applicable additional agreements.\n If you have a valid license agreement with SAP for the use of a particular SAP\n External Product, then you may make use of any API Calls included in this\n project's code for that SAP External Product, subject to the terms of such\n license agreement. If you do not have a valid license agreement for the use of\n a particular SAP External Product, then you may only make use of any API Calls\n in this project for that SAP External Product for your internal, non-productive\n and non-commercial test and evaluation of such API Calls. Nothing herein grants\n you any rights to use or access any SAP External Product, or provide any third\n parties the right to use of access any SAP External Product, through API Calls."
 
 [[annotations]]
 path = [
   ".github/CODEOWNERS",
   ".github/renovate.json",
   ".gitignore",
+  ".goreleaser.yaml",
   ".license-scan-overrides.jsonl",
   ".license-scan-rules.json",
+  "CONTRIBUTING.md",
+  "Dockerfile",
   "go.mod",
   "go.sum",
   "Makefile.maker.yaml",
+  "main.go",
+  "README.md",
+  "REUSE.toml",
   "vendor/modules.txt",
+  "internal/**",
+  "docs/**"
```

Comment:
Note for us: I fixed the licenses headers of most files and added them to the missing ones. goreleaser was removed as it is unused and the Dockerfile we add in the templating.